pkg/ovsdb/handler.go

```diff
@@ -210,7 +243,12 @@ func (ch *Handler) MonitorCondChange(ctx context.Context, param []interface{}) (
 
 func (ch *Handler) MonitorCondSince(ctx context.Context, param ovsjson.CondMonitorParameters) (interface{}, error) {
 	klog.V(5).Infof("MonitorCondSince request, parameters %v", param)
-
+	ch.mu.Lock()
+	defer ch.mu.Unlock()
+	if ok := ch.createOrDieMonitorsArray(param.JsonValue.(string)); ok != nil {
+		return nil, fmt.Errorf("duplicate json-value")
+	}
+	ch.jsonValueToUpdateNotificationType[param.JsonValue.(string)] = ovsjson.Update3
 	data, err := ch.getMonitoredData(param.DatabaseName, param.MonitorCondRequests, true)
 	if err != nil {
 		return nil, err
```

Comment:
you need to cleanup `ch.jsonValueToUpdateNotificationType` if out due to error

Reply:
yes, I have to clean up this one and set and clean up `jsonValueToMonitors`.
This part is not done yet, I will work on it today.